internal/generator/vector/output/http/http.go

```diff
@@ -184,14 +185,19 @@ func Headers(o logging.OutputSpec) Element {
 }
 
 func ToHeaderStr(h map[string]string) string {
+	sortedKeys := make([]string, 0, len(h))
+	for k := range h {
+		sortedKeys = append(sortedKeys, k)
+	}
+	sort.Strings(sortedKeys)
 	str := ""
 	skipComma := true
-	for k, v := range h {
+	for _, k := range sortedKeys {
 		if skipComma {
-			str += fmt.Sprintf("%q=%q", k, v)
+			str += fmt.Sprintf("%q=%q", k, h[k])
```

Comment:
What about using `strings.Join` in multiple iterations so there is no need to "skip" the comma or not?

Reply:
updated.